memory_reference/README.md

```diff
@@ -0,0 +1 @@
+# Memory Reference Project
```

Comment:
Please write a small description, what the mem_ref class is doing how to use.

Document also, how to enable and disable the CUDA GPU build.